### ISSUE TITLE
Demo Site?

### ISSUE THREAD

**Issue by kaoskeya** [kaoskeya/meteor-admin#8]
Will be nice to have a demo site on meteor or elsewhere.

**Comment by livvyjeffs** [comment]
Agreed! I have been trying to use houston:admin and also kaoskeya:admin and can't get the user database in either one. How do I publish? Are they already published? How do I use this dburles:collection-helper function? This would really help, I feel quite a bit lost.
